### DISCUSSION TITLE
Charging current rate.

### DISCUSSION THREAD

**Discussion by Thesizeofya** [Hyundai-Kia-Connect/kia_uvo#599] [Ideas]
Would it be possible to start a charge with different charging current rates. It would be handy for solar generation.

**Comment by cdnninja** [comment]
Is it in the mobile app?

**Reply by Thesizeofya** [reply]
I can't see it, you do have a setting in the car. **Charging Current**  Maximum(7.4kw) /Reduced 6kw/Minimum (4kw).

**Reply by cdnninja** [reply]
If not offered in the native app not something we can do.

**Reply by Thesizeofya** [reply]
ok, thanks for looking, I can always set the car charge rate to minimum and start/stop the charge remotely when the Solar produces more than 4kw.
